lom/src/lib/lomcommon/yang_parser/yang_prod_configs/device-health-procs-configs.yang

```diff
@@ -0,0 +1,47 @@
+module device-health-procs-configs {
```

Comment:
YANG models should be in a dedicated folder outside src.
Currently there is a YANG models dir in root folder.
You may move these there.
Or have a dedicated YANG folder under lom/ folder.

Parser only a tool.
Over time, YANG Models will be the one taking more frequent changes. 
Being action definitions we need to move it in front.

Suggest 
remove current yang-models and create one under lom/